projects/storefrontlib/shared/components/captcha/captcha.component.html

```diff
@@ -0,0 +1,2 @@
+<div #captcha class="g-recaptcha"></div>
+<cx-form-errors [control]="control"></cx-form-errors>
```

Comment:
Lets not have `cx-form-errors` here within the template to allow the user to add the errors tag where they like externally (eg. under the input like on the registration form).